packages/dashboards/run-dev.sh

```diff
@@ -54,7 +54,11 @@ export OPENCRVS_METABASE_MAP_NAME=${OPENCRVS_METABASE_MAP_NAME:-"Farajaland"}
 export OPENCRVS_METABASE_MAP_URL=${OPENCRVS_METABASE_MAP_URL:-"http://localhost:3040/content/farajaland-map.geojson"}
 export OPENCRVS_METABASE_MAP_REGION_KEY=${OPENCRVS_METABASE_MAP_REGION_KEY:-"State"}
 export OPENCRVS_METABASE_MAP_REGION_NAME=${OPENCRVS_METABASE_MAP_REGION_NAME:-"State"}
-
+export OPENCRVS_METABASE_ADMIN_EMAIL=${OPENCRVS_METABASE_ADMIN_EMAIL:-"superuser@opencrvs.org"}
```

Comment:
Let's keep the defaults as the defaults documented in the readme @tahmidrahman-dsi

Reply:
thanks Riku, done 60c2f859e9c65525af6dd1bac6e92c8e1464c82c